app/views/provider_interface/reasons_for_rejection/edit_other_reasons.html.erb

```diff
@@ -1,33 +1,33 @@
 <% content_for :before_content, govuk_back_link_to(provider_interface_reasons_for_rejection_initial_questions_path(@application_choice)) %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <span class="govuk-caption-l"><%= @application_choice.application_form.full_name %></span>
-    <h1 class="govuk-heading-l">Other reasons for rejection</h1>
-    <p class="govuk-body">The candidate will see this feedback.</p>
-  </div>
   <div class="govuk-grid-column-two-thirds">
     <%= form_with model: @wizard, url: provider_interface_reasons_for_rejection_update_other_reasons_path do |f| %>
 
       <%= f.govuk_error_summary %>
 
+      <span class="govuk-caption-l"><%= @application_choice.application_form.full_name %></span>
+
       <% if @wizard.reason_not_captured_by_initial_questions? %>
+        <h1 class="govuk-heading-l">Other reasons for rejecting this application</h1>
```

Comment:
This copy change (and others in the flow) have come from the prototype, which I assume is what we want